test/scraper.e2e-spec.ts

```diff
@@ -0,0 +1,33 @@
+import { INestApplication } from "@nestjs/common";
+import { Test } from "@nestjs/testing";
+import { DepositFixture } from "../src/modules/scraper/adapter/db/deposit-fixture";
+import { AppModule } from "../src/app.module";
+import { QueryFailedError } from "typeorm";
+
+describe("Scraper module", () => {
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleFixture = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
```

Comment:
This app instance can then be used to test endpoints with things like [supertest](https://github.com/visionmedia/supertest) ?

Reply:
Exactly! It'd be smth like

```js
import request from "supertest";

const response = await request(app.getHttpServer()).get("url");
expect(response.data)...
expect(response.statusCode)...
```

[NestJS docs example](https://docs.nestjs.com/fundamentals/testing#end-to-end-testing)